src/storage/page_buffer.c

```diff
@@ -6913,11 +6910,6 @@ pgbuf_search_hash_chain (THREAD_ENTRY * thread_p, PGBUF_BUFFER_HASH * hash_ancho
 		  goto two_phase;
 		}
 
-	      if (loop_cnt++ < mbw_cnt)
```

Comment:
The following things can be romoved:
- int loop_cnt;
- mutex_lock:
- mutex_lock2:

Reply:
OK, I removed it